ts/features/itwallet/issuance/screens/ItwIssuanceEidPreviewScreen.tsx

```diff
@@ -0,0 +1,117 @@
+import {
+  ButtonLink,
+  ButtonSolid,
+  ContentWrapper,
+  IOColors,
+  VSpacer
+} from "@pagopa/io-app-design-system";
+import * as O from "fp-ts/lib/Option";
+import { pipe } from "fp-ts/lib/function";
+import React from "react";
+import { StyleSheet, View } from "react-native";
+import { OperationResultScreenContent } from "../../../../components/screens/OperationResultScreenContent";
+import { IOScrollViewWithLargeHeader } from "../../../../components/ui/IOScrollViewWithLargeHeader";
+import I18n from "../../../../i18n";
+import { useIONavigation } from "../../../../navigation/params/AppParamsList";
+import { EidCardPreview } from "../../common/components/EidCardPreview";
+import { ItwCredentialClaimsList } from "../../common/components/ItwCredentialClaimList";
+import {
+  ItWalletError,
+  getItwGenericMappedError
+} from "../../common/utils/itwErrorsUtils";
+import { ItwCredentialsMocks } from "../../common/utils/itwMocksUtils";
+import { StoredCredential } from "../../common/utils/itwTypesUtils";
+
+export const ItwIssuanceEidPreviewScreen = () => {
+  const navigation = useIONavigation();
+  const eidOption = O.some(ItwCredentialsMocks.eid);
+
+  /**
+   * Renders the content of the screen if the PID is decoded.
+   * @param eid - the decoded eID
+   */
+  const ContentView = ({ eid }: { eid: StoredCredential }) => {
+    React.useLayoutEffect(() => {
+      navigation.setOptions({
+        headerShown: true
+      });
+    }, []);
+
+    return (
+      <IOScrollViewWithLargeHeader
+        title={{
+          label: I18n.t("features.itWallet.issuance.credentialPreview.title", {
+            credential: eid.displayData.title
+          })
+        }}
+      >
+        <View style={styles.preview}>
+          <EidCardPreview />
+        </View>
+        <View style={styles.dropShadow}>
+          <VSpacer size={24} />
+        </View>
+        <View style={styles.content}>
+          <ItwCredentialClaimsList data={eid} />
+        </View>
+        <ContentWrapper>
+          <VSpacer size={32} />
+          <ButtonSolid
+            label={I18n.t(
+              "features.itWallet.issuance.credentialPreview.actions.primary"
+            )}
+            icon="add"
+            iconPosition="end"
+            onPress={() => undefined}
+            fullWidth={true}
+          />
+          <VSpacer size={24} />
+          <View style={{ alignSelf: "center" }}>
+            <ButtonLink
+              label={I18n.t(
+                "features.itWallet.issuance.credentialPreview.actions.secondary"
+              )}
+              onPress={() => undefined}
+            />
+          </View>
+        </ContentWrapper>
```

Comment:
There is a FooterStackButtons (deprecated) that adds footer with two vertically stacked buttons. Btw in the deprecation the hint is to use GradientScrollView. @dmnplb is this the right solution to get vertically stacked buttons?

Reply:
We could wait for the merge of this https://github.com/pagopa/io-app/pull/5787

Reply:
@hevelius @mastro993 Better use the new `FooterActions` in this case (which is visually similar to the old `FooterStackButtons`)

Reply:
> @hevelius @mastro993 Better use the new `FooterActions` in this case (which is visually similar to the old `FooterStackButtons`)

`FooterActions` does not work properly with the `IOScrollViewWithLargeHeader`. I think it's better to merge this PR and then think about a proper solution later.